filament/backend/src/opengl/platforms/PlatformEGLHeadless.cpp

```diff
@@ -138,6 +138,17 @@ backend::Driver* PlatformEGLHeadless::createDriver(void* sharedContext) noexcept
         goto error;
     }
 
+    // fallback to a 24-bit depth buffer
+    if (mEGLConfig == nullptr) {
```

Comment:
I think it would be slightly better to check `configsCount==0` instead

Reply:
Done.